pydantic_ai_slim/pydantic_ai/models/bedrock.py

```diff
@@ -116,12 +119,30 @@
 T = typing.TypeVar('T')
 
 
-class BedrockModelSettings(ModelSettings):
+class BedrockModelSettings(ModelSettings, total=False):
     """Settings for Bedrock models.
 
     ALL FIELDS MUST BE `bedrock_` PREFIXED SO YOU CAN MERGE THEM WITH OTHER MODELS.
     """
 
+    bedrock_guardrail_config: GuardrailConfigurationTypeDef
+    """Content moderation and safety settings for Bedrock API requests."""
+
+    bedrock_performance_configuration: PerformanceConfigurationTypeDef
+    """Performance optimization settings for model inference."""
+
+    bedrock_request_metadata: dict[str, str]
+    """Additional metadata to attach to Bedrock API requests."""
+
+    bedrock_additional_model_response_fields_paths: list[str]
+    """JSON paths to extract additional fields from model responses."""
+
+    bedrock_prompt_variables: Mapping[str, PromptVariableValuesTypeDef]
+    """Variables for substitution into prompt templates."""
+
+    bedrock_additional_model_requests_fields: Mapping[str, Any]
+    """Additional model-specific parameters to include in requests."""
```

Comment:
It would be nice to add links to those.

```suggestion
    bedrock_additional_model_requests_fields: Mapping[str, Any]
    """Additional model-specific parameters to include in requests.
    
    See more about it on <url>.
    """
```

Reply:
Yes, especially since you deal with dozens of integrations, this concern is super valid.